dot/rpc/modules/grandpa_integration_test.go

```diff
@@ -37,21 +37,18 @@ func TestGrandpaProveFinality(t *testing.T) {
 	testStateService.Block.SetJustification(bestBlock.Header.Hash(), make([]byte, 11))
 
 	var expectedResponse ProveFinalityResponse
-	expectedResponse = append(expectedResponse, make([]byte, 10), make([]byte, 11))
+	expectedResponse = append(expectedResponse, "0x0000000000000000000000")
 
 	res := new(ProveFinalityResponse)
 	err = gmSvc.ProveFinality(nil, &ProveFinalityRequest{
-		blockHashStart: bestBlock.Header.ParentHash,
-		blockHashEnd:   bestBlock.Header.Hash(),
+		BlockNumber: uint32(bestBlock.Header.Number),
 	}, res)
 
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	if !reflect.DeepEqual(*res, expectedResponse) {
-		t.Errorf("Fail: expected: %+v got: %+v\n", res, &expectedResponse)
-	}
+	assert.Equal(t, expectedResponse, *res)
```

Comment:
nit/ a bit out of scope: you should assert pointer vs pointer (so change expectedResponse to be a pointer), in case `res` is nil, to avoid panicing.